test/clojure/nrepl/bencode_test.clj

```diff
@@ -206,3 +205,31 @@
       (is (thrown? IllegalArgumentException
                    (write-bencode out {"obj" (Object.)})))
       (is (= "d3:obj" (String. (.toByteArray out)))))))
+
+(def valid-bencode-input-generator
```

Comment:
You might add some section comment before the `test.check` tests, so they are easier to spot when someone's scanning the file.